src/index.js

```diff
@@ -3,13 +3,25 @@ import ReactDOM from "react-dom";
 import "./index.css";
 import App from "./App";
 import { makeServer } from "./server";
+import { BrowserRouter } from "react-router-dom";
+import { WatchLaterProvider } from "./Context/WatchLaterContext";
+import { LikedVideosProvider } from "./Context/LikedVideosContext";
+import { VideoProvider } from "./Context/videoContext";
```

Comment:
you can create an `index.js` file in the context folder and import all of them in this single file. Importing them here and writing these context imports in a single line make the code look clean!